src/pages/academy/grading/subcomponents/GradingWorkspace.tsx

```diff
@@ -308,7 +308,7 @@ class GradingWorkspace extends React.Component<GradingWorkspaceProps, State> {
             xpAdjustment={props.grading![questionId].grade.xpAdjustment}
             maxXp={props.grading![questionId].question.maxXp}
             studentName={props.grading![questionId].student.name}
-            comments={props.grading![questionId].grade.comments!}
+            comments={props.grading![questionId].grade.comments || ''}
```

Comment:
```suggestion
            comments={props.grading![questionId].grade.comments ?? ''}
```

Should we make use of the nullish coalescing operator instead to better capture the semantics here (i.e., use the empty string if `comments` is `undefined` or `null` instead of the broader "is falsey")?

Reply:
Thanks for the suggestion! I agree!